File: Ad-Hoc/pagina09/1893/1893.php
<?php 

    list($porcent1, $porcent2) = explode(" ", readline());

    $porcent1 = (int) $porcent1;
    $porcent2 = (int) $porcent2;

    if ( $porcent2 <= 2 && $porcent2 >= 0 )
        echo "nova\n";

    else if ( $porcent2 <= 100 && $porcent2 >= 97 )
        echo "cheia\n";

    else if ( $porcent1 < $porcent2)
        echo "crescente\n";

    else 
        echo "minguante\n"; 
?>

File: Ad-Hoc/__1000__/1893/1893.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {

        Scanner in = new Scanner(System.in);
        String[] input = in.nextLine().split(" ");

        int porcent1 = Integer.parseInt(input[0]);
        int porcent2 = Integer.parseInt(input[1]);

        if ( porcent2 <= 2 && porcent2 >= 0 )
            System.out.println("nova");

        else if ( porcent2 <= 100 && porcent2 >= 97 )
            System.out.println("cheia");

        else if ( porcent1 < porcent2)
            System.out.println("crescente");

        else 
            System.out.println("minguante"); 

        in.close();
    }
}